### PR TITLE
Branch income category bug fix

### PR DESCRIPTION
**Pull request by NeoMinWei**
Fix assertion bug

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM! Appreciate the quick response and bug fix

**Review by wwweert123 — APPROVED** [review]
Change Noted

File: src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java [hunk 1]
```diff
@@ -54,22 +54,22 @@ private void handleInvalidCategory(String stringCategory) {
...
-        assert index != 0;
-
         if (category == null) {
             handleDeleteCashflowWithoutCategory();
             return;
         }
-        Ui ui = Ui.getInstance();
+
+        assert category.equals(CashflowCategory.INCOME) || category.equals(CashflowCategory.EXPENSE);
+        assert index != 0;
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java:63]: Good use of assertions!

**Review by wwweert123 — COMMENTED** [review]
👍

File: src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java (hunk 1 above)
Comment (NeoMinWei) [reply src/main/java/seedu/financialplanner/commands/DeleteCashflowCommand.java:63]: thanks!

**Pull request merged**